tests/unit/disco/sagas/testDiscovery.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
+
+import { loadEntities } from 'core/actions';
+import { ErrorHandler } from 'core/errorHandler';
+import addonsReducer from 'core/reducers/addons';
+import apiReducer from 'core/reducers/api';
+import * as api from 'disco/api';
+import { getDiscoResults, loadDiscoResults } from 'disco/actions';
+import discoResultsReducer from 'disco/reducers/discoResults';
+import discoverySaga from 'disco/sagas/discovery';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import {
+  createFetchDiscoveryResult, fakeDiscoAddon,
+} from 'tests/unit/disco/helpers';
+
+
+describe('disco/sagas/discovery', () => {
```

Comment:
Hmmm, interesting. Fair enough, that's a good point. You can search the project for the text in the spec, but I see the value of this for sure.

Works for me!